index.html

```diff
@@ -332,17 +332,16 @@
           descriptor.
         </p>
         <p>
-          If the descriptor does not have a deviceId, its semantic is that
-          it queries for access to all devices of that class. Thus, if a query
-          for the "camera" permission with no deviceId returns "granted", the
+          If the descriptor does not have a deviceId, its semantic is that it
+          queries for access to all devices of that class. Thus, if a query for
+          the "camera" permission with no deviceId returns "granted", the
           client knows that there will never be a permission prompt for a
```

Comment:
Because https://github.com/w3c/permissions/pull/69 hasn't been merged yet, and GitHub does a bad job presenting series of dependent patchsets. Each of these PRs actually has a single commit, so you could just review https://github.com/w3c/permissions/pull/70/commits/68ab77e2e99c4fcfdef28d21e678b6f5facdcf4d.